run_test.go

```diff
@@ -112,6 +112,27 @@ func TestRunsCreate(t *testing.T) {
 		assert.Equal(t, cvTest.ID, r.ConfigurationVersion.ID)
 	})
 
+	t.Run("with refresh not set", func(t *testing.T) {
```

Comment:
Excuse the nit, but could we change this to something like `"with refresh not set as a create option, it defaults to true"`

Reply:
Good callout for our future selves trying to remember why this got tested separately 😆